### ISSUE TITLE
Weird note rendering

### ISSUE THREAD

**Issue by 9662** [stefan-niedermann/OwnCloud-Notes#77]
Take this note:
[Shopping List.txt](https://github.com/stefan-niedermann/OwnCloud-Notes/files/113782/Shopping.List.txt)

On my phone it renders like this:
![screenshot_2016-02-02-12-09-12](https://cloud.githubusercontent.com/assets/13080040/12747728/14a97f2a-c9a6-11e5-8f0f-106cbf23c180.png)

Any idea what I'm doing wrong? Am I just crap at markdown formatting or is this a bug? Cheers.

Just to clarify, I do not expect checkboxes. It is the line breaks that I'm concerned about.

**Comment by stefan-niedermann** [comment]
This is a known bug in the upstream-library. See #24 "Line breaks ignored" and the correspondent bug in the upstream project "bypass": https://github.com/Uncodin/bypass/issues/197

Sadly, no one seems to care... :-/

**Comment by stefan-niedermann** [comment]
Try to use 
```
Shopping List


- [ ] Avocado 2
- [ ] Champignons
- [ ] Pepper
- [ ] Tomatoes
- [ ] Tuna
- [ ] Olives (green)
- [ ] Raw ham
- [ ] Dry mozzarella no need
- [ ] Basilicum

- [ ] Fruit juice
- [ ] Pine nuts
- [ ] Water

```

This could work.

**Comment by 9662** [comment]
I am pleased to announce that this is, after all, [not a bug](https://daringfireball.net/projects/markdown/syntax#block). It seems that Markdown does not take into account single newlines, thus catering for hard-wrapped text.

However, it will honour any single newline preceded by at least two spaces. If I take my own shopping list example and modify it like so (note the two spaces at the end of each line, except the ‘basilicum’ one):
![screenshot_20160209_111901](https://cloud.githubusercontent.com/assets/13080040/12914359/2660c394-cf24-11e5-81db-ff4eb284159c.png)

The result is like this:
![screenshot_2016-02-09-11-17-58](https://cloud.githubusercontent.com/assets/13080040/12914396/68b8131e-cf24-11e5-89b5-3fd5bcde5660.png)

Quod volebamus est. :smiley:

**Comment by stefan-niedermann** [comment]
Oh yeah! Nice!! Thank you. This will fix maybe #24.

**Comment by 9662** [comment]
Or just chalk this down as a feature, not a bug? After all, the only problem (in my case) was that I hadn't read the Markdown spec and went on what Github does instead.

**Comment by stefan-niedermann** [comment]
Well that's just marketing ;) The current behavior is not the expected, so it's a bug for me. This fix works and will be in v0.7.0.

Thank you very much for the hint.
